app/controllers/api/comments_controller.rb

```diff
@@ -26,4 +26,8 @@ def update_mobile_user_identity
     MobileUserIdentity.new(@mobile_user).update(first_name: params.dig(:first_name),
                                                 last_name: params.dig(:last_name))
   end
+
+  def comments
+    current_debate.moderate? ? current_debate.comments.active : current_debate.comments
+  end
```

Comment:
Maybe we should move this logic to debate so it will check its own state and decide which comments to return